### ISSUE TITLE
Wrong emoji loading

### ISSUE THREAD

**Issue by corradopetrelli** [DomeT99/my-website#3]
### Description

Emojis are not displayed on the blog page

![image](https://github.com/DomeT99/astro-minimal/assets/36475130/5f25e975-9e9c-4f38-b353-76c510799953)

**Comment by DomeT99** [comment]
Thank you @corradopetrelli for this issue.
I will try to resolve the problem as soon as possible.
In the meantime, feel free to open a PR :)

**Comment by corradopetrelli** [comment]
I tried by phone and it doesn't give me the same error.
Very strange that it doesn't work on macOS.

Some emojies do not work 😢 

![image](https://github.com/DomeT99/astro-minimal/assets/36475130/018500d8-e0ba-468f-8cc6-3798e0c64166)

**Comment by DomeT99** [comment]
Do you use Safari on MacOS?

**Comment by corradopetrelli** [comment]
No, I use Chrome 117 on MacOS. It's definitely an OS problem since it doesn't show me the emoji even on the notepad.
